interpreter/cling/tools/plugins/clad/CMakeLists.txt

```diff
@@ -59,13 +73,9 @@ ExternalProject_Add(
              -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
              -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
              -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
-             -DCMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD}
              -DCMAKE_INSTALL_PREFIX=${clad_install_dir}/plugins
-             -DLLVM_DIR=${LLVM_DIR}
-             -DClang_DIR=${Clang_DIR}
```

Comment:
@vgvassilev do we need Clang_DIR still?

Reply:
Yes, we need it conditionally. I thought clad improved its automatic detection of this but apparently not...